city_scrapers/spiders/chi_ohare_noise.py

```diff
@@ -0,0 +1,167 @@
+                links=self._parse_links(item),
+                start=self._parse_start(item),
+                source=self._parse_source(response)
+            )
+
+            meeting["status"] = self._get_status(meeting)
+            meeting["id"] = self._get_id(meeting)
+            yield meeting
+            self.meetings[url_date] = meeting
+        curr_date = datetime.date.today()
+        months_urls = [datetime.date(month=curr_date.month-2, day=1, year=curr_date.year),
+                       datetime.date(month=curr_date.month-1, day=1, year=curr_date.year),
+                       curr_date,
+                       datetime.date(month=curr_date.month+1, day=1, year=curr_date.year)
+                       ]
+        for date in months_urls:
+            yield Request(
+                    "https://www.oharenoise.org/about-oncc/oncc-meetings/month.calendar/" +
+                    date.strftime("%Y/%m/%d"), callback=self._parse_calendar, priority=1)
+
+    def _parse_months(self):
+        curr_date = datetime.date.today()
+        months_urls = [datetime.date(month=curr_date.month-2, day=1, year=curr_date.year),
+                       datetime.date(month=curr_date.month-1, day=1, year=curr_date.year),
+                       curr_date,
+                       datetime.date(month=curr_date.month+1, day=1, year=curr_date.year)
+                       ]
+        for date in months_urls:
+            yield Request(
+                    "https://www.oharenoise.org/about-oncc/oncc-meetings/month.calendar/" +
+                    date.strftime("%Y/%m/%d"), callback=self._parse_calendar, priority=1)
+
+    def _parse_calendar(self, response):
+        event_urls = response.selector.xpath(".//a[@class='cal_titlelink']//@href").extract()
+        for event_url in event_urls:
+            yield Request("https://www.oharenoise.org" + event_url, self._parse_event)
+
+    def _parse_event(self, response):
+        info_body = response.selector.xpath(".//div[@id='jevents_body']")
+        name_date_time = info_body.xpath(".//div[@class='jev_evdt_header']")
+        start = self._parse_start(name_date_time, is_calen=True)
+        date_key = start.strftime("%Y/%m/%d")
+        description = self._parse_description(info_body)
+        location = self._parse_location(info_body)
+        meeting = None
+        if date_key in self.meetings:
+            meeting = self.meetings[date_key]
+            meeting['start'] = start
+            meeting['description'] = description
+            meeting['location'] = location
+        else:
+            meeting = Meeting(
+                    title=self._parse_title(name_date_time, is_calen=True),
+                    start=start,
+                    description=description,
+                    location=location
+                    )
+            self.meetings[date_key] = meeting
+        yield meeting
+
+    def _not_empty(self, string):
+        if string is not None and not str.isspace(string):
```

Comment:
Empty strings and `None` evaluate to `False`, so instead of having a function for this you can just use `if string and string.strip():`